### PR TITLE
docs(split): fix deployment documentation for missing command

### PR DESCRIPTION
**Pull request by iadjadj**
Ensure proper functionality by fixing missing command in deployment documentation

### CODE REVIEW

**Review by esoubiran-aneo — APPROVED** [review]

File: .docs/content/1.installation/1.linux/1.installation.md
Comment (pmilaneo) [inline .docs/content/1.installation/1.linux/1.installation.md:76]:
```suggestion
If you do not create a Kubernetes namespace, the deployment will fail.
```

File: .docs/content/1.installation/1.linux/1.installation.md
Comment (pmilaneo) [inline .docs/content/1.installation/1.linux/1.installation.md:79]:
```suggestion
Now, you can deploy each resource independently :
```

**Review by pmilaneo — CHANGES_REQUESTED** [review]
a few changes and it's ok

**Review by lzianekhodja — APPROVED** [review]

**Review by pmilaneo — CHANGES_REQUESTED** [review]
there is line 38 to change into
If you do not create the data directory, the deployment will fail when the compute plane tries to mount the shared directory.
sorry, I missed it during the first review

**Review by lzianekhodja — DISMISSED** [review]

**Comment by lzianekhodja** [comment]
> there is line 38 to change into If you do not create the data directory, the deployment will fail when the compute plane tries to mount the shared directory. sorry, I missed it during the first review

Yes off course, this is what was explained in the old documentation in Markdown files at the root of the deployment project.

**Review by pmilaneo — CHANGES_REQUESTED** [review]
how that should be good

**Review by lzianekhodja — APPROVED** [review]

**Comment by aneojgurhem** [comment]
@iadjadj can you resolve the conflict ?
@pmilaneo you should approve when the conflict are resolved because your requested changes are blocking the merge

**Review by pmilaneo — DISMISSED** [review]

**Pull request merged**
